legend-engine-xts-analytics/legend-engine-xts-analytics-search/legend-engine-xt-analytics-search-pure/src/main/resources/core_analytics_search/trans.pure

```diff
@@ -114,7 +114,7 @@ function meta::analytics::search::transformation::dataspace::buildExecutionConte
 
 function meta::analytics::search::transformation::dataspace::buildClassDocuments(mapping: meta::pure::mapping::Mapping[1]): meta::analytics::search::metamodel::class::SimpleClassElement[*]
 {
-  let analysedMapping = meta::analytics::mapping::modelCoverage::analyze($mapping, true, true);
+  let analysedMapping = meta::analytics::mapping::modelCoverage::analyze($mapping, true, true, false);
```

Comment:
why do we do this?

Reply:
I added a new boolean to mapping analysis whether to analyze the mapped classes or not